.eslintrc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   parserOptions: {
     project: './tsconfig.lint.json',
+    warnOnUnsupportedTypeScriptVersion: false,
```

Comment:
Disables the huge TS version warning. Docs: https://typescript-eslint.io/packages/parser/#warnonunsupportedtypescriptversion